### PR TITLE
EVAKA-4406 Add utilization to info in units list

### PR DESCRIPTION
**Pull request by zch**
- Refactor the unit list view in the employee mobile frontend to look like the designs
- Add the current unit utilization to the info shown

### CODE REVIEW

File: service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt [hunk 1]
```diff
@@ -180,20 +196,20 @@ WITH present_children AS (
     ) p
     GROUP BY p.unit_id
 ), present_staff AS (
-    SELECT g.daycare_id AS unit_id, sum(sa.count) AS count
+    SELECT g.daycare_id AS unit_id, sum(sa.capacity) AS capacity, sum(sa.count) AS count
     FROM daycare_group g
     JOIN (
-        SELECT sa.group_id, sa.count + sa.count_other AS count FROM staff_attendance sa
+        SELECT sa.group_id, 7 as capacity, sa.count + sa.count_other AS count FROM staff_attendance sa
```
Comment (jaahon) [inline service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt:202]: Shouldn't `capacity` be something like `7 * (sa.count + sa.count_other)`?

**Review by jaahon — COMMENTED** [review]
Apart from the staff capacity thing looks good to me!

File: service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt (hunk 1 above)
Comment (zch) [reply service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt]: Great catch, thanks!

While testing I decided to add handling to return Infinity utilization if there are kids but no staff (should not happen, but might if someone forgets to add the staff).

File: service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt [hunk 2]
```diff
@@ -210,7 +226,11 @@ SELECT
     coalesce(pc.count, 0) AS present_children,
     coalesce(tc.count, 0) AS total_children,
     coalesce(ps.count, 0) AS present_staff,
-    coalesce(ts.count, 0) AS total_staff
+    coalesce(ts.count, 0) AS total_staff,
+    CASE
+        WHEN coalesce(ps.capacity, 0) = 0 THEN 'Infinity'::REAL
```
Comment (jaahon) [inline service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt:231]: Infinity is now shown even if there are no children present. Is that intentional?

**Review by jaahon — APPROVED** [review]

**Pull request merged**